vectordraw/templates/html/vectordraw.html

```diff
@@ -0,0 +1,130 @@
+{% load i18n %}
+<div class="vectordraw_block">
+
+  <h2>{{ self.display_name }}</h2>
+
+  {% if self.description %}
+    <div class="vectordraw-description">
+      {{ self.description|safe }}
+    </div>
+  {% endif %}
+
+  <div id="vectordraw">
+    <div class="menu" style="width: {{ self.width }}px;">
+      <div class="controls">
+        <span class="sr" id="element-list-add-label">{% trans "Select element to add to board" %}</span>
```

Comment:
@itsjeyd can we make this a label element and change the aria-labelledby attribute below to a for attribute?  It's better supported and won't trigger test failures.